README.md

```diff
@@ -201,12 +200,6 @@ $ k3d cluster delete tns
 $ rm -rf tanka
 ```
 
-## Troubleshooting
-
-**Issue:** 404 error when trying to load Tempo traces.
-
-**Solution:** Your Jaeger agent is likely not running correctly. Check that all pods were successfully scheduled.
```

Comment:
I don't see any "Jaeger agent" so this step is likely obsolete. Alternatively we could keep it and say "Grafana agent" instead.